docs/source/release_notes/release-notes.rst

```diff
@@ -48,24 +74,12 @@ v0.14.0
            This will only affect deployments who run with more than one server (i.e. in HA mode).
            (`Pull Request <https://github.com/palantir/atlasdb/pull/873>`__)
 
-    *    - |breaking|
+    *    - |fixed|
```

Comment:
Is this intentional?  (I agree "fixed" is a better description because the old configuration wouldn't have worked anyway - but wanted to double-check)